.github/workflows/build_apps.yml

```diff
@@ -62,7 +63,12 @@ jobs:
             else
               echo "build_from=${{inputs.version}}" | tee -a $GITHUB_OUTPUT
             fi
-            echo "Build from: "
+            if [[ "${{inputs.version}}" == "" || "${{ github.ref_name }}" == "liya/DEV-2307-updates-for-helm-chart" ]]; then
```

Comment:
Why there's a reference to your branch.

Reply:
Thanks :) need to change to main, was set to my branch for tests..